huxley/core/admin/assignment.py

```diff
@@ -59,7 +59,8 @@ def generate_assigments(reader):
                     rejected = False # allow for the rejected field to be null
                 else:
                     rejected = (row[3].lower() == 'true') # use the provided value if admin provides it
-                yield (committee.id, country.id, school.id, rejected)
+                is_double = True
```

Comment:
This is wrong. What this will do is set every single assignment to have the 'is_double' field as True. This isn't the behavior we want because not every assignment is a double delegation. What probably needs to happen is we need to change our CSV schema for uploading assignments to have a is_double field. We can then set the is_double field to whatever value is passed in by external.